apps/sms/services/test/unit/moz_mobile_message/moz_mobile_message_shim_test.js

```diff
@@ -254,7 +265,14 @@ suite('MozMobileMessageShim >', function() {
       test('continue', function() {
         MockNavigatormozMobileMessage.getThreads.returns(cursor);
         MozMobileMessageShim.getThreads(streamStub);
-        cursor.result = {};
+        cursor.result = {
```

Comment:
nit: maybe we can move the result and continue stub in cursor setup since they are reused between 2 tests?